src/Content/Widget.php

```diff
@@ -125,6 +125,15 @@ public static function unavailableNetworks()
 		if (!Addon::isEnabled("pnut")) {
 			$networks[] = Protocol::PNUT;
 		}
+		return $networks;
+	}
+
+	/**
+	 * Return unavailable networks
+	 */
+	public static function unavailableNetworks()
```

Comment:
```suggestion
	public static function unavailableNetworksCondition()
```
Why not return a `[condition, parameter]` condition?

Reply:
I wanted to make this the most generic usable at the moment. We can always change this anytime if needed.